packages/node-resolve/package.json

```diff
@@ -75,6 +75,11 @@
       "!**/types.ts"
     ]
   },
-  "module": "dist/index.es.js",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/es/index.js"
+  },
+  "module": "./dist/es/index.js",
+  "type": "commonjs",
```

Comment:
Should this say `commonjs` instead of `module`? When I was reading https://nodejs.org/api/esm.html#esm_conditional_exports I saw that they use `type` `module` together with `exports` `require` and `import`.

Reply:
For modules there is package.json with {type:module} in dist/es

Reply:
Interesting. Is there a reason a nested `package.json` approach is preferred to declaring everything in the top-level `package.json`?

Reply:
I think `type: commonjs` in a package.json is not really necessary as this is the default anyway: https://nodejs.org/api/esm.html#esm_package_json_type_field

But we can still keep it for clarity's sake.